PeriodicMeetings/2023-07-17.md

```diff
@@ -0,0 +1,60 @@
+
+[Link to last meeting notes](./2023-07-10.md)<!-- Edit this link to the date of the last meeting -->
+
+#### B
+
+- [X] Flesh out the "Comparison With Other Packages/Ecosystems" section (later today - 2023-07-20)
+    - [X] BioJulia vs BioPython
+    - [X] BioJulia vs BioConductor
+- [X] Generate basic outline (what's included in "Getting Started",...) (later today - 2023-07-20)
+
+#### Other
+
+- [X] Generate Bowtie/Bowtie2 JLL package 
+    - [X] Generate Bowtie2 build recipe (https://github.com/JuliaPackaging/Yggdrasil/pull/7051)
+    - [X] Generate Bowtie (v1) build recipe (https://github.com/JuliaPackaging/Yggdrasil/pull/7069)
+- [X] Gather feedback for other packages to be ported to BinaryBuilder/Yggdrasil for BioJulia (https://julialang.slack.com/archives/CAKKFNYLD/p1689685204319719)
+
+
+### Notes
+
+- Presently gearing up for JuliaCon so am tackling some smaller projects this week that can be finished before I leave on July 24
+- Should arrive in Boston on the afternoon of July 24
+
+## What I plan to do this week
+
+<!-- Note: For longer-term goals or tasks, also add to project notes or to appropriate project repository -->
+- As I am gearing up for JuliaCon, will look to address some minor items including further ports and generating more ideas for BioJuliaDocs
+- I looked into the Franklin.jl source code and I think there is an opportunity for a first pull request:
+    - Currently, Franklin.jl relies on [html-css-js-minify](https://github.com/juancarlospaco/css-html-js-minify/tree/master) which requires Python3 as a dependency
+    - [I recently ported Minify](https://github.com/JuliaPackaging/Yggdrasil/pull/7067) which is a minifier written in Go. It can work on more files than the Python3 package (JSON, SVG, and XML files), can be built into a single executable without any external dependency, and runs [much faster](https://github.com/tdewolff/minify#time-lower-is-better)
+    - My pull request would focus on replacing html-css-js-minify with the minify_jll package and thus eliminate the Python3 dependency as well as grant additional minification functionality 
+
+#### Other
+
+- [] Port IQ-Tree to Yggdrasil (requires Eigen3, Zlib)
+    - [] Generate IQ-Tree build recipe
+- [] Port KMA Aligner to Yggdrasil
+    - [] Generate KMA Aligner build recipe (requires Zlib)
+- [] Franklin.jl pull request
+    - [] Add minify_jll dependency
+    - [] Remove html-css-js-minify, .py files
+    - [] Update documentation
+    - [] Submit pull request
+
+## Is there anything that Kevin can help with?
+
+- I would like to setup a project board [as discussed on Slack](https://julialang.slack.com/archives/CAKKFNYLD/p1689703789780049?thread_ts=1689685204.319719&cid=CAKKFNYLD) so the BioJulia community can better track package ports. I am not a member of the org on GitHub so I need an organizer to create one/grant me permission to work on the project board
+- Let me know when would be best to meet you/Guilherme in Boston :)
```

Comment:
Let's grab lunch on Wednesday?

Reply:
Sure! Lunch time is between 12:30 and 1:30. Where should we meet?